crates/extensions/c8y_mapper_ext/src/operations/handlers/mod.rs

```diff
@@ -258,6 +263,7 @@ fn to_c8y_operation(operation_type: &OperationType) -> Option<CumulocitySupporte
         OperationType::SoftwareList => None,
         // local-only operation, not always invoked by c8y, handled in other codepath
         OperationType::Health => None,
+        OperationType::DeviceProfile => Some(CumulocitySupportedOperations::C8yDeviceProfile),
```

Comment:
Minor: Moving this line between `OperationType::SoftwareUpdate` and `OperationType::SoftwareList` will look better. Then the operation types at upper part return `Some(_)`, and others at bottom return `None`.